src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeController.php

```diff
@@ -134,16 +136,38 @@ public function updatePositionAction(Request $request, int $attributeGroupId)
      *     message="You do not have permission to create this."
      * )
      *
-     * @param int $attributeGroupId
+     * @param Request $request
      *
-     * @return RedirectResponse
+     * @return Response
      */
-    public function createAction(int $attributeGroupId)
+    public function createAction(Request $request, int $attributeGroupId): Response
     {
-        // @todo: implement in another pr
-        return $this->redirectToRoute('admin_attributes_index', [
-            'attributeGroupId' => $attributeGroupId,
-        ]);
+        $attributeFormBuilder = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeFormBuilder');
+        $attributeFormHandler = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\AttributeFormHandler');
```

Comment:
```suggestion
        $attributeFormBuilder = $this->get(AttributeFormBuilder::class);
        $attributeFormHandler = $this->get(AttributeFormHandler::class);
```
You should use the class static const instead of a hard-coded string

Reply:
Actually there is no `AttributeFormBuilder` or `AttributeFormHandler` class, it's a service definition based on `PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\FormBuilder`

Reply:
If there is no `AttributeFormBuilder` then we shouldn't use a service name that looks like an FQCN it's confusing, we should a use a text service name instead matching the namespace convention like other similar services: `prestashop.core.form.identifiable_object.builder.attribute_form_builder`

Same thing for the form handler